### PR TITLE
RNs 39668 + 39636 + 39604 + 39682 + 39625

### PR DESCRIPTION
**Pull request by EdithLansens**
@WoutMH and @stephenlanszweertsky, could you please review this pull request to confirm the accurate inclusion of release notes 39668, 39636, 39604, 39682, and 39625 in the user guide? Thanks in advance!

### CODE REVIEW

File: user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Low_Code_Apps/Low-Code_Apps_configuration/LowCodeApps_event_config.md [hunk 1]
```diff
@@ -6,7 +6,23 @@ uid: LowCodeApps_event_config
 
 You can configure actions for different types of events in the DataMiner Low-Code Apps. These are the main types:
 
-- *On page load*: This event takes place when a page is loaded. (See [Configuring a page of a low-code app](xref:LowCodeApps_page_config).)
+- *On open*: This event takes place when a page or panel is opened. (See [Configuring a page of a low-code app](xref:LowCodeApps_page_config) and [Configuring a panel of a low-code app](xref:LowCodeApps_panel_config).)
```
Comment (WoutMH) [inline user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Low_Code_Apps/Low-Code_Apps_configuration/LowCodeApps_event_config.md:9]: when a page or panel is opened and all components are done loading

**Review by WoutMH — COMMENTED** [review]
one small comment

File: user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Low_Code_Apps/Low-Code_Apps_configuration/LowCodeApps_event_config.md (hunk 1 above)
Comment (EdithLansens) [reply user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Low_Code_Apps/Low-Code_Apps_configuration/LowCodeApps_event_config.md:9]:
```suggestion
- *On open*: This event takes place when a page or panel is opened and all components are done loading. (See [Configuring a page of a low-code app](xref:LowCodeApps_page_config) and [Configuring a panel of a low-code app](xref:LowCodeApps_panel_config).)
```

File: user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Low_Code_Apps/Low-Code_Apps_configuration/LowCodeApps_page_config.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ To configure a page:
 
 1. Click the pencil icon to configure the components on the page. See [Configuring components](xref:Configuring_components) and [Changing dashboard settings](xref:Changing_dashboard_settings). For a list of the available components, refer to [Available visualizations](xref:Available_visualizations).
 
-1. If one or more actions should be triggered when the page is loaded, in the page configuration pane, open the *Events* section and click the configuration button next to *On page load*. Then configure the action(s) that should be triggered. See [Configuring low-code app events](xref:LowCodeApps_event_config).
+1. If one or more actions should be triggered when the page is loaded or closed, in the page configuration pane, open the *Events* section and click the *Configure actions* button next to *On open* (or *On page load*) or *On close* (available from DataMiner 10.3.0 [CU16]/10.4.0 [CU4]/10.4.7 onwards<!--RN 39604 + 39682-->). Then configure the action(s) that should be triggered. See [Configuring low-code app events](xref:LowCodeApps_event_config).
```
Comment (MariekeGO) [inline user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Low_Code_Apps/Low-Code_Apps_configuration/LowCodeApps_page_config.md:23]: Perhaps we could leave out the version info here but immediately link to a place where this info can be found, to make this easier to read?
```suggestion
1. If one or more actions should be triggered when the page is loaded or closed, in the page configuration pane, open the *Events* section and click the *Configure actions* button next to [*On open* (or *On page load*) or *On close*](xref:LowCodeApps_event_config). Then configure the action(s) that should be triggered.
```

Comment (EdithLansens) [reply user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Low_Code_Apps/Low-Code_Apps_configuration/LowCodeApps_page_config.md]: Agreed!

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**